tests/contrib/unittest/test_unittest_patch.py

```diff
@@ -3,7 +3,7 @@
 # removed the ``_generated`` suffix from the file name, to prevent the content
 # from being overwritten by future re-generations.
 
-from ddtrace.contrib.unittest import get_version
+from ddtrace.contrib.unittest.patch import get_version
```

Comment:
```suggestion
from ddtrace.contrib.internal.unittest.patch import get_version
```

Reply:
Unittest is not in `ddtrace.contrib.internal`. None of the ci vis stuff was internalized in 2.X.